OpenRA.Mods.Common/EditorBrushes/EditorActorBrush.cs

```diff
@@ -99,10 +99,10 @@ public AddActorAction(EditorActorLayer editorLayer, ActorReference actor)
 
 		public void Execute()
 		{
-			Do();
+			Do(false);
 		}
 
-		public void Do()
+		public void Do(bool isUndoRedoAction)
```

Comment:
`isUndoRedoAction` is overkill. Can't it be handled by Execute or something like that?

Reply:
Turns out a separate event triggered upon user actions was the way to go. This is also triggered by `ClearSelection` if the optional argument `updateSelectedTab` is set to true.